### DISCUSSION TITLE
Unable to update column to NULL through Swift Library

### DISCUSSION THREAD

**Discussion by vahidg** [supabase/supabase#29253] [Questions] [answered]
I cannot set the value of a field to NULL through the Swift Library. The column is nullable. 

Here's my update statement:

```
                let response = try await supabase.from("projects")
                    .update(UpdateProject(logoUrl: nil))
                    .eq("id", value: editedProject.id!)
                    .execute()
```

Here's the UpdateProject struct:

```
struct UpdateProject: Encodable {
        var logoUrl: String?
    
        enum CodingKeys: String, CodingKey {
            case logoUrl = "logo_url"
        }
}
```

If I pass a string instead of `nil`, then the field is successfully changed. But with `nil`, the old value (url) remains.

The request always returns 200.

I'm wondering, if there is a way for me to see the actual query being run as a result of these queries, but haven't found a way to see that through the logs.

Any help would be greatly appreciated.

**Comment by GaryAustin1** [comment, marked as answer]
https://github.com/supabase/supabase-swift/issues/218
https://github.com/supabase/supabase-swift/issues/203
https://github.com/supabase/supabase-swift/issues/93

I don't use Swift, but several asked and seemingly answered null questions.

**Reply by vahidg** [reply]
Thank you, indeed the answer was to be found in those threads. I should have searched better!
